File: Java/Easy/WrongAnswer/minimum-amount-of-time-to-fill-cups_0.java
class Solution {
    public int fillCups(int[] amount) {
        
        int cold = amount[0];
        int warm = amount[1];
        int hot = amount[2];
        int time = 0;
        while (cold > 0 || warm > 0 || hot > 0) {
            if (cold > 0 && warm > 0) {
                cold--;
                warm--;
                time++;
            } else if (warm > 0 && hot > 0) {
                warm--;
                hot--;
                time++;
            } else if (hot > 0 && cold > 0) {
                cold--;
                hot--;
                time++;
            } else if (cold > 0) {
                cold--;
                time++;
            } else if (warm > 0) {
                warm--;
                time++;
            } else if (hot > 0) {
                hot--;
                time++;
            }
        }
        return time;
    }
}

File: Python3/Easy/RuntimeError/minimum-amount-of-time-to-fill-cups_0.py
class Solution:
    def fillCups(self, amount: List[int]) -> int:
        
        def get_problem_level(all_problems, problem_name):
            for level, problems in all_problems.items():
                if problem_name in problems:
                    return level
            return -1
        return get_problem_level(all_problems, "fill-cups")